File: app/src/main/java/com/smedialink/abakarmagomedov/githubapplication/presentation/UsersActivity/UsersInteractor.java
package com.smedialink.abakarmagomedov.githubapplication.presentation.UsersActivity;


import com.smedialink.abakarmagomedov.githubapplication.data.entity.User;

import java.util.List;

import io.reactivex.Observable;

/**
 * Created by abakarmagomedov on 15/05/17.
 */

public interface UsersInteractor {
    Observable<List<User>> loadUsersFromGit();
}

File: app/src/main/java/com/smedialink/abakarmagomedov/githubapplication/presentation/UsersActivity/UsersInteractor.kt
package com.smedialink.abakarmagomedov.githubapplication.presentation.UsersActivity

import com.smedialink.abakarmagomedov.githubapplication.data.entity.User
import io.reactivex.Observable

/**
 * Created by abakarmagomedov on 15/05/17.
 */
interface UsersInteractor {
    fun loadUsersFromGit(): Observable<List<User>>
}
